lm_eval/tasks/drop.py

```diff
@@ -50,19 +51,34 @@ def _load_docs(self, docs):
                     "id": qa["query_id"],
                     "passage": doc["passage"],
                     "question": qa["question"],
-                    "answers": self.get_answers(qa["answer"]),
+                    "answers": self.get_answers(qa),
                 }
 
     @classmethod
-    def get_answers(cls, answers):
-        # NOTE: We wrap every non-`list` answer into a list for uniformity.
-        if answers["number"] != "":
-            return [str(answers["number"])]
-        if answers["spans"] != []:
-            return answers["spans"]
-        return [" ".join([answers["date"]["day"],
-                          answers["date"]["month"],
-                          answers["date"]["year"]]).strip()]
+    def get_answers(cls, qa):
+        answers = []
+        answers_set = set()
+
+        candidates = [qa["answer"]] + qa.get("validated_answers", [])
+        for candidate in candidates:
+            answer = cls.parse_answer(candidate)
+            if answer in answers_set:
+                continue
+            answers_set.add(answer)
+            answers.append(answer)
+
+        return answers
```

Comment:
They don't seem to "unique-ify" the `candidates` [here](https://github.com/allenai/allennlp-models/blob/5012f2387decc806152fcba6ad81345b7627fc2a/allennlp_models/rc/tools/drop.py#L213). You can remove this class method and create a local variable in `_load_docs` like `candidate_answers = [qa["answer"]] + qa.get("validated_answers", [])`. Then set the `answers` doc entry value to be the list comprehension result of calling `parse_answer` across candidate answers.